sdk/core/core/src/az_base64.c

```diff
@@ -159,10 +161,10 @@ AZ_NODISCARD az_result az_base64_encode(
       assert(sxt + TRIBYTE_SEXTET2_INDEX < result_size);
       buffer.begin[sxt + TRIBYTE_SEXTET2_INDEX] = (oct + TRIBYTE_OCTET1_INDEX < input.size)
           ? uint6_as_base64(
-              base64url,
```

Comment:
this should be an if statement, this is a somewhat insane ternary imho. No change requested for this PR.